publisher-sdk/src/androidTest/java/com/criteo/publisher/CriteoBannerViewIntegrationTest.java

```diff
@@ -49,13 +50,22 @@ public void setup() throws Exception {
 
     runOnMainThreadAndWait(() -> {
       criteoBannerView = new CriteoBannerView(
-          InstrumentationRegistry.getContext(),
+          InstrumentationUtil.getApplication().getApplicationContext(),
```

Comment:
Nit: as you have access to the mocked dependency provide, you might get a context by adding `@Inject private lateinit var context: Context` in the field section.